test/reuse.jl

```diff
@@ -0,0 +1,71 @@
+# Depending on how the internals of `@patch` and `apply` are setup there could be issues
+# with reusing a patch for a variety of reasons. These were discovered with experimenting
+# with using Cassette with Mocking.
```

Comment:
Does the history of these tests matter?

Reply:
The history can be useful for determining if these tests can be removed in the future